rust/examples/video-tx.rs

```diff
@@ -0,0 +1,95 @@
+use anyhow::{bail, Context, Result};
+use std::net::Ipv4Addr;
+use std::rc::Rc;
+use std::sync::atomic::{AtomicBool, Ordering};
+use std::sync::Arc;
+
+use imtl::mtl::{Flags, LogLevel, MtlBuilder};
+use imtl::netdev::*;
+use imtl::session::RtpSessionBuilder;
+use imtl::video::{Fps, TransportFmt, VideoTxBuilder};
+
+fn main() -> Result<()> {
+    let running = Arc::new(AtomicBool::new(true));
+    let r = running.clone();
+    ctrlc::set_handler(move || {
+        r.store(false, Ordering::SeqCst);
+    })
+    .expect("Error setting Ctrl-C handler");
+
+    /* open a yuv file from disk and map to memory */
+    let yuv_file = std::fs::File::open("/tmp/test.yuv").context("Failed to open yuv file")?;
```

Comment:
nit: use arguments to pass the yuv source file.

Reply:
Thanks, done.